### PR TITLE
Tidied up multiple scripts, added one and outright deleted one.

### PR DESCRIPTION
**Pull request by ghost**
Changes made:

- Added a DB management/lib function script;

- Deleted the filter/iterate script; (there are reliable native methods for doing this)

- Tidied up function parameters, some code, colors, comments and ~~readme typos~~ made the whole repository be compatible with the new subscript notation

### CODE REVIEW

**Comment by HeatherSoron** [comment]
@A-Casual-Guy Is "low-dash parameter" common jargon anywhere that you've seen? I ask about that, specifically, because Lodash is a fairly common library outside of Hackmud.

File: scripts/char_lister.js [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
-		return{
+	if (!a){
+		return {
 			ok: false,
-			msg: "Define how many !characters! you want to display (The list starts at 01)."
+			msg: "Define how many `Ncharacters` you want to display (The list starts at 01)."
 		}
 	}
 	//iterates through a for loop, converting the numbers to characters
+	//note - for (let foo; foo<bar.length; foo++) syntax is less readable than a for...in loop - changed it
```
Comment (HeatherSoron) [inline scripts/char_lister.js]: The original one wasn't `a.characters.length`, so this is actually different functionality. It actually looks like the original was the intended form, in this case?

File: scripts/lib_functions_and_database_mgmnt.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	switch (a.f) {
+		case "to_gc":
+			 return "\n`AInput: " + a.f + "`\n" + "`AOutput: " + l.to_gc_str(a.args) + "`"
+		case "from_gc":
+			return "\n`AInput: " + a.f + "`\n" + "`AOutput: " + l.to_gc_num(a.args) + "`"
+		case "timestr":
+			return "\n`A"+l.to_game_timestr(new Date())+"`"
+		case "i":
+			#db.i(a.args)
+			return #db.f(a.args)
```
Comment (HeatherSoron) [inline scripts/lib_functions_and_database_mgmnt.js:44]: Missing `.first()` or `.array()`?

**Review by HeatherSoron — COMMENTED** [review]
I'm not sold on stripping `context` from scripts with no/minor changes. It is useful as a placeholder name, IMO. I might go with similar logic there to indentation changes, or other (non-functionality-changing) style fixups - if you're not touching other stuff, don't aggressively normalize.

File: scripts/lib_functions_and_database_mgmnt.js (hunk 2 above)
Comment (ghost) [reply scripts/lib_functions_and_database_mgmnt.js:44]: Whoops. I'll fix this up.

**Pull request closed without merging**